src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -480,7 +486,10 @@ function unbound_add_host_entries($ifconfig_details = null)
             if (isset($config['dhcpdv6'][$interface]['enable']) && !empty($config['dhcpdv6'][$interface]['domain'])) {
                 $domain = $config['dhcpdv6'][$interface]['domain'];
             }
-            $unbound_entries .= "local-data-ptr: \"{$laddr6} {$config['system']['hostname']}.{$domain}\"\n";
+            if ($interface === $interfaces[0] && !in_array($laddr6, $ptr_records, true)) {
```

Comment:
lo0 or similar may throw this one off

Reply:
Isn't this caught by https://github.com/opnsense/core/blob/2e42838751cf2c60aab605b00bf315452afe8744/src/etc/inc/plugins.inc.d/unbound.inc#L466-L468?

Reply:
fair point, let me check which interfaces could end up there or if this is already good

Reply:
ipsec devices throw this off as well, see other comment and commit